app/models/pawn.rb

```diff
@@ -6,4 +6,39 @@ def image
       '&#9823;'
     end
   end
+
+  def valid_move?(x, y)
```

Comment:
Rubocop thinks this method is too complex.  Which in general it's a good idea to keep methods simple and split them into multiple methods if they get too involved, but in this case I think it makes sense to keep everything here (the goal is readability, and this is very readable).  Can you add the following to the bottom of the file named `.rubocop.yml`?  This should relax that complexity check a bit.

```yaml
Metrics/CyclomaticComplexity:
  Max: 10
```